packages/rancher-monitoring/66.7/rancher-grafana/.gitignore

```diff
@@ -1 +1,2 @@
-charts
\ No newline at end of file
+./charts
```

Comment:
BTW, I realized I did this wrong in hindsight.

I corrected them (properly this time) in: https://github.com/rancher/ob-team-charts/pull/62